src/routes/loginRoute.ts

```diff
@@ -1,8 +1,8 @@
 import Router from 'express'
-
 import LoginController from '../controllers/LoginController'
+import { loginValidator } from '../middlewares/routeValidation'
 
 const router = Router()
-router.post('/login', LoginController.login)
+router.post('/login', loginValidator, LoginController.login)
 
```

Comment:
Try to put this login into user Route. we don't need to have many route which is  common. i.e. If you are working with a user, we have use Route which has all endpoints of user like signup, login, get user ,delete User, verifyEmail etc . Which is the same as for products and so on.